edudoc/staff-crash-course/staff_crash_course.md

```diff
@@ -0,0 +1,94 @@
+
+More broadly speaking, the preferred  approach is “over-communication” in the WCA by keeping all relevant persons in the loop at all times. If you think there is a topic which other staff members should see, please add them to as a Carbon Copy (CC) to the email. For example, if you escalate an issue about a specific Delegate to any Team/Committee, be sure to include their Senior and/or Regional Delegate in CC.
+
+<div style="page-break-after: always; visibility: hidden"> \pagebreak </div>
+
+## Making changes to recipients
+
+When you make changes to the recipients, make this clear at the top of the email.\
+If there is a need to include another team/committee to the thread then begin by writing a “+” along with the name of the team/committee so everyone is aware.\
+When removing someone from a thread they were once a part of you write a “-” followed by their name.\
+
+	*For Example:*
+	(+WQAC -WRC)
+
+	**Your email would go here**
+
+If a change is needed and the original recipient needs to see it has gone through and has been taken care of, they can be moved to BCC, allowing them to see the last email go through. After that, they will not receive further emails on the thread.
+
+Lastly, for team/committee internal discussion you can take a shortcut by removing everyone but your team/committee’s email address and write “*[TEAM] Only*”.
+
+## Representation
+
+Every WCA Staff member will also have received login credentials for their own personal *@worldcubeassociation.org* email account hosted by GSuite.\
+This account must always be used when sending WCA-related emails. Exceptions may be made only if the Staff Member has issues with the GSuite Software in the region where they live.
+
+Configure your new GSuite account to include a signature that comprises your name and your position (and nothing else, like 3x3 PBs or YouTube channels!). Be aware that this signature will be visible to **any recipient, along with a legal disclaimer for recipients outside of the** *@worldcubeassociation.org* domain.
+
+Think twice before sending any form of private messages through this account. You should never use this email for any work that is not related to the WCA. Please refer to the [WCA Email Account Policy](https://www.worldcubeassociation.org/documents/policies/internal/Email%20Account.pdf) for details.
+
+## Mailing Lists
+
+There are several mailing lists which all have their dedicated audience and purposes. All team and committee addresses can be found on the [contact page of the WCA](https://www.worldcubeassociation.org/contact), along with a brief description of their scope of work.\
+Other significant mailing lists include:
+
+  - [Reports@worldcubeassociation.org](mailto:Reports@worldcubeassociation.org) - Recipient channel for all Delegate reports, worldwide. You can opt-in on your WCA profile settings page.
```

Comment:
Can we please use lowercase for these emails?
While it should not matter for gmail, maybe people have set up filters that are case sensitive and I'd rather be careful with that.